src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  background-color: #fff;
```

Comment:
isn't it white by default?

Reply:
sh*****t. Forgot about that. I'm using background colors for a better vision of the elements.